### ISSUE TITLE
Getting Internal Server Error while stamping cid

### ISSUE THREAD

**Issue by sami-colon** [rigwild/ipfs-ark-stamp#1]
I received this json response when i tried to use stamp feature.

Response {bodyJson: {…}, type: "basic", url: "http://localhost:5000/ark/broadcastCid/QmQmLzw2bqg1qbVAz8vf9iiNpsTtuUSnYSZuGNuyg7cknY", redirected: false, status: 500, …}
body: (...)
bodyJson:
message: "Not Found"
__proto__: Object
bodyUsed: true
headers: Headers {}
ok: false
redirected: false
status: 500
statusText: "Internal Server Error"
type: "basic"
url: "http://localhost:5000/ark/broadcastCid/QmQmLzw2bqg1qbVAz8vf9iiNpsTtuUSnYSZuGNuyg7cknY"

**Comment by rigwild** [comment]
Try adding the following code here to get the raw output of the ARK node API: https://github.com/rigwild/ipfs-ark-stamp/blob/master/server/index.ts#L297

Then check the Node.js console.

```ts
console.error(JSON.stringify(result, null, 2))
```

___

Note: If you are here because of this listing: 
 - https://www.freelancer.com/projects/html/Files-transfert-via-Blockchain/
 - https://www.freelancer.com/projects/software-architecture/Ark-Blockchain-Expert-Needed/

Respect carefully the license terms.

**Comment by sami-colon** [comment]
yes this is giving correct result 
console.error(JSON.stringify(result, null, 2))

but it is giving this error 
script.js:193 Error: Cannot read property 'body' of undefined
    at script.js:78
    at async wn.broadcastCid (script.js:186)

**Comment by rigwild** [comment]
Without the correct line in the ts file I cannot help you. Please give more details. This is not helpful at all.

**Comment by sami-colon** [comment]
I tried to console out 
console.error(JSON.stringify(result, null, 2))
and it is the result I got when i tried stamping.

`{                                                                                                                               "body": {                                                                                                                       "data": {                                                                                                                       "accept": [],                                                                                                                 "broadcast": [],                                                                                                              "excess": [],                                                                                                                 "invalid": [                                                                                                                    "6e7b18a77bb4ceba46647b30fa4334333439559df6bfaff5e6f9872ae5ff3555"                                                          ]                                                                                                                           },                                                                                                                            "errors": {                                                                                                                     "6e7b18a77bb4ceba46647b30fa4334333439559df6bfaff5e6f9872ae5ff3555": [                                                           {                                                                                                                               "type": "ERR_APPLY",                                                                                                          "message": "Failed to apply transaction, because this IPFS hash is already registered on the blockchain."                   }                                                                                                                           ]                                                                                                                           }                                                                                                                           },                                                                                                                            "headers": {},                                                                                                                "status": 200                                                                                                               }                                                                                                                             TypeError: Cannot read property 'body' of undefined `

and then when i tried uploading new picture and then tried stamping i received this error.
`{                                                                                                                               "body": {                                                                                                                       "data": {                                                                                                                       "accept": [                                                                                                                     "0072e975f9f51ef40cfa68d875c25032e3d4476eb7683940cd5114c0e47eec6e"                                                          ],                                                                                                                            "broadcast": [                                                                                                                  "0072e975f9f51ef40cfa68d875c25032e3d4476eb7683940cd5114c0e47eec6e"                                                          ],                                                                                                                            "excess": [],                                                                                                                 "invalid": []                                                                                                               }                                                                                                                           },                                                                                                                            "headers": {},                                                                                                                "status": 200                                                                                                               }                                                                                                                             TypeError: Cannot read property 'body' of undefined     `

Actually it is getting this body error in script.js file line no 78.


Here are ark.ts and script.js
https://pastebin.com/MHNbSBk9
https://pastebin.com/utyfnyjD
